spec/support/models/typecasted_author.rb

```diff
@@ -1,8 +1,8 @@
 class TypecastedAuthor < ::ActiveRemote::Base
-  attribute :guid, :type => String
-  attribute :name, :typecaster => StringTypecaster
-  attribute :age, :type => Integer
-  attribute :birthday, :type => DateTime
-  attribute :writes_fiction, :type => Boolean
-  attribute :net_sales, :type => Float
+  attribute :guid, :string
```

Comment:
Would it be worth adding a compatibility wrapper to map `:type => String` to `:string`, etc?

Reply:
I'm just looking at that because we're using the attribute definition stuff in Atlas, and upgrading is going to be a major pain. I'm actually looking at doing an intermediate release to allow us to use this new interface with the existing codebase so we can update Atlas accordingly.

Reply:
Chatted through this with @newellista offline, and we think doing an intermediate release will be a little more work, but make the transition a lot smoother. Going to hold off on merging this until I can get that sorted out.